docs/production/dbos-cloud/secrets.md

```diff
@@ -1,23 +1,26 @@
 ---
 sidebar_position: 100
-title: Secrets and Environment Variables
+title: Environment Variables
```

Comment:
I'd keep the secrets name, and emphasize these are stored securely.

Reply:
yeah, tho I'd use the `env` CLI sub command